scripts/bootstrap/config.js

```diff
@@ -0,0 +1,7 @@
+var config = {};
```

Comment:
I think a JSON format is more easy to read, but I do not have a strong preference, leave it to you.